shopfloor/tests/test_actions_data.py

```diff
@@ -156,14 +161,14 @@ def test_data_move_line_package(self):
             "package_src": {
                 "id": move_line.package_id.id,
                 "name": move_line.package_id.name,
-                "move_line_count": 1,
+                "move_line_count": 0,
```

Comment:
That means `self.move_a` is `done`, `cancel` or `confirmed` here?

Reply:
Added a check for the state in these tests.